common/src/main/java/com/wynntils/screens/WynntilsDialogueHistoryScreen.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.mc.objects.CommonColors;
+import com.wynntils.mc.render.FontRenderer;
+import com.wynntils.mc.render.HorizontalAlignment;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.render.TextRenderSetting;
+import com.wynntils.mc.render.TextRenderTask;
+import com.wynntils.mc.render.Texture;
+import com.wynntils.mc.render.VerticalAlignment;
+import com.wynntils.screens.widgets.BackButton;
+import com.wynntils.screens.widgets.PageSelectorButton;
+import com.wynntils.screens.widgets.QuestsPageButton;
+import com.wynntils.screens.widgets.ReloadButton;
+import com.wynntils.utils.MathUtils;
+import com.wynntils.wynn.event.QuestBookReloadedEvent;
+import com.wynntils.wynn.model.questbook.QuestBookManager;
+import java.util.ArrayList;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.components.AbstractButton;
+import net.minecraft.client.gui.components.Widget;
+import net.minecraft.client.gui.components.events.GuiEventListener;
+import net.minecraft.client.resources.language.I18n;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class WynntilsDialogueHistoryScreen extends WynntilsMenuPagedScreenBase {
```

Comment:
Nice to see something similar to `QuestBookListPage` here

Reply:
Yeah, it is really basic though, but I like that about this. If we do item guides, it will be easy.